apps/common-app/src/apps/reanimated/examples/ShadowNodesCloningExample.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useMemo, useState } from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import Animated, { useSharedValue, withTiming } from 'react-native-reanimated';
+
+function AnimatedView() {
+  const opacitySv = useSharedValue(0);
+  const scaleSv = useSharedValue(0);
+  const [counter, setCounter] = useState(0);
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      opacitySv.value = withTiming(Math.random(), { duration: 300 });
+      scaleSv.value = withTiming(Math.random() * 2, { duration: 300 });
```

Comment:
Wouldn't it be better if we use `withRepeat` instead of `setInterval` for running the animations? Does it affect the repro in any way?